lib/Cake/Utility/Inflector.php

```diff
@@ -495,7 +495,12 @@ public static function underscore($camelCasedWord) {
  */
 	public static function humanize($lowerCaseAndUnderscoredWord) {
 		if (!($result = self::_cache(__FUNCTION__, $lowerCaseAndUnderscoredWord))) {
-			$result = ucwords(str_replace('_', ' ', $lowerCaseAndUnderscoredWord));
+			$result = str_replace('_', ' ', $lowerCaseAndUnderscoredWord);
+			if (function_exists('mb_convert_case') && Multibyte::checkMultibyte($result)) {
```

Comment:
Given that the inputs to humanize() are documented to be `lower_case_and_underscore` we should probably look at updating the tests.

I don't like having the code behave differently if the function & data are different because it makes the result of the the method more difficult to predict. Having the method be as consistent as possible is more important to me, as it makes the behavior clearer and more transparent.